### PR TITLE
Install Insight using open-release/ginkgo.master version 

### PR DESCRIPTION
**Pull request by RizRiyz**
Updated the branch `open-release/ginkgo.master` to support insight installation on ginkgo.master version. 

I've successfully install inisight on Ubuntu 16.04 by using this branch in the following script:
https://github.com/valutac/vania/blob/master/install-insights.sh

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ariestiyansyah! I've created [OSPR-2597](https://openedx.atlassian.net/browse/OSPR-2597) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: playbooks/roles/analytics_api/defaults/main.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ ANALYTICS_API_DATABASES:
     PORT: '3306'
     OPTIONS: "{{ ANALYTICS_API_REPORTS_MYSQL_OPTIONS }}"
 
-ANALYTICS_API_VERSION: "master"
+ANALYTICS_API_VERSION: "open-release/ginkgo.master"
```
Comment (nedbat) [inline playbooks/roles/analytics_api/defaults/main.yml]: Why does this need to be changed here?  In Hawthorn, this still says "master".

**Comment by mduboseedx** [comment]
@ariestiyansyah Thank you for the pull request. Should a similar change also be made on the hawthorn.master branch?

**Comment by RizRiyz** [comment]
@mduboseedx yes, I'll update on hawthorn too.

**Comment by RizRiyz** [comment]
Made a PR for `open-release/hawthorn.master` #4761

**Comment by nedbat** [comment]
Wait wait wait: I want to understand why this change is here. It's not the way we've been making changes for Ginkgo and Hawthorn.

**Comment by RizRiyz** [comment]
@nedbat I found some error when install the insight on ginkgo.master branch, it direct me to the master branch of edx-analytics-api and insights which lead me to some error, after made change the version of `INSIGHT_VERSION`, `ANALYTCIS_API_VERSION` and `sqoop` path my installation is running well.

So I made my own script here: https://github.com/valutac/vania/blob/master/install-insights.sh

**Comment by RizRiyz** [comment]
@nedbat wait wait, are you guys using `INSIGHT_VERSION` and `ANALYTCIS_API_VERSION` in  `--extra-vars` parameter?

**Comment by nedbat** [comment]
@ariestiyansyah in native.sh, we set INSIGHTS_VERSION and ANALYTICS_API_VERSION to OPENEDX_RELEASE.  Shouldn't this take care of it?

**Comment by RizRiyz** [comment]
@nedbat Yeah! I remember if we can use that in ansible, okay I've update my PR to revert the value to `master`, but sqoop path still need to update :) you can review it again.

I also updated my vania :), yeayyy! : https://github.com/valutac/vania/blob/master/install-insights.sh

**Review by nedbat — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@ariestiyansyah 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/4760&entry.1484655318=https://github.com/ariestiyansyah&entry.752974735=2018-09-06+15:13&entry.1917517419=2018-09-07+20:57&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**